public/keymaps/dztech_dz65rgb_default.json

```diff
@@ -0,0 +1 @@
+{"keyboard":"dztech/dz65rgb","keymap":"dztech/dz65rgb_layout_65_ansi_mine","layout":"LAYOUT_65_ansi","layers":[["KC_GESC","KC_1","KC_2","KC_3","KC_4","KC_5","KC_6","KC_7","KC_8","KC_9","KC_0","KC_MINS","KC_EQL","KC_BSPC","KC_HOME","KC_TAB","KC_Q","KC_W","KC_E","KC_R","KC_T","KC_Y","KC_U","KC_I","KC_O","KC_P","KC_LBRC","KC_RBRC","KC_BSLS","KC_PGUP","LCTL_T(KC_CAPS)","KC_A","KC_S","KC_D","KC_F","KC_G","KC_H","KC_J","KC_K","KC_L","KC_SCLN","KC_QUOT","KC_ENT","KC_PGDN","KC_LSFT","KC_Z","KC_X","KC_C","KC_V","KC_B","KC_N","KC_M","KC_COMM","KC_DOT","KC_SLSH","KC_RSFT","KC_UP","KC_END","KC_LCTL","KC_LGUI","KC_LALT","KC_SPC","KC_RALT","MO(1)","KC_RCTL","KC_LEFT","KC_DOWN","KC_RGHT"],["KC_ESC","KC_F1","KC_F2","KC_F3","KC_F4","KC_F5","KC_F6","KC_F7","KC_F8","KC_F9","KC_F10","KC_F11","KC_F12","KC_DEL","KC_NO","KC_NO","RGB_TOG","RGB_MOD","RGB_HUI","RGB_HUD","RGB_SAI","RGB_SAD","RGB_VAI","RGB_VAD","KC_NO","KC_PSCR","KC_SLCK","KC_PAUS","RESET","KC_NO","KC_NO","RGB_SPI","RGB_SPD","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","EEP_RST","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_VOLU","KC_MUTE","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_TRNS","KC_NO","KC_MPRV","KC_VOLD","KC_MNXT"]]}
```

Comment:
This suggestion is what I would *prefer*, but as I said on the other PR, I can't think of a situation where a user who changes between Configurator and compiling from source would be confused by this. This is quite likely just me being extremely particular.

The suggestion:

- turns all the `KC_NO` keycodes to `KC_TRNS`, as the keys are assigned in qmk_firmware's default keymap
- changes `KC_GESC` to `KC_ESC` on the function layer
- adds `LCTL_T(KC_CAPS)`, Left Shift, Home, Page Up and Page Down keys to the function layer (same as qmk_firmware `make`)

```suggestion
{"keyboard":"dztech/dz65rgb","keymap":"dztech/dz65rgb_layout_65_ansi_mine","layout":"LAYOUT_65_ansi","layers":[["KC_GESC","KC_1","KC_2","KC_3","KC_4","KC_5","KC_6","KC_7","KC_8","KC_9","KC_0","KC_MINS","KC_EQL","KC_BSPC","KC_HOME","KC_TAB","KC_Q","KC_W","KC_E","KC_R","KC_T","KC_Y","KC_U","KC_I","KC_O","KC_P","KC_LBRC","KC_RBRC","KC_BSLS","KC_PGUP","LCTL_T(KC_CAPS)","KC_A","KC_S","KC_D","KC_F","KC_G","KC_H","KC_J","KC_K","KC_L","KC_SCLN","KC_QUOT","KC_ENT","KC_PGDN","KC_LSFT","KC_Z","KC_X","KC_C","KC_V","KC_B","KC_N","KC_M","KC_COMM","KC_DOT","KC_SLSH","KC_RSFT","KC_UP","KC_END","KC_LCTL","KC_LGUI","KC_LALT","KC_SPC","KC_RALT","MO(1)","KC_RCTL","KC_LEFT","KC_DOWN","KC_RGHT"],["KC_GESC","KC_F1","KC_F2","KC_F3","KC_F4","KC_F5","KC_F6","KC_F7","KC_F8","KC_F9","KC_F10","KC_F11","KC_F12","KC_DEL","KC_HOME","KC_TRNS","RGB_TOG","RGB_MOD","RGB_HUI","RGB_HUD","RGB_SAI","RGB_SAD","RGB_VAI","RGB_VAD","KC_TRNS","KC_PSCR","KC_SLCK","KC_PAUS","RESET","KC_PGUP","LCTL_T(KC_CAPS)","RGB_SPI","RGB_SPD","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","EEP_RST","KC_PGDN","KC_LSFT","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_VOLU","KC_MUTE","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_MPRV","KC_VOLD","KC_MNXT"]]}
```